easybuild/easyconfigs/s/segmentation-models-pytorch/segmentation-models-pytorch-0.3.3-foss-2022a.eb

```diff
@@ -0,0 +1,49 @@
+easyblock = 'PythonBundle'
+
+name = 'segmentation-models-pytorch'
+version = '0.3.3'
+
+homepage = 'https://github.com/qubvel/segmentation_models.pytorch'
+description = "Python library with Neural Networks for Image Segmentation based on PyTorch."
+
+toolchain = {'name': 'foss', 'version': '2022a'}
+
+dependencies = [
+    ('Python', '3.10.4'),
+    ('PyTorch', '1.12.0'),
+    ('torchvision', '0.13.1'),
+    ('Safetensors', '0.3.1'),
+]
+
+use_pip = True
+
+exts_list = [
+    ('munch', '2.5.0', {
+        'checksums': ['2d735f6f24d4dba3417fa448cae40c6e896ec1fdab6cdb5e6510999758a4dbd2'],
+    }),
+    ('pretrainedmodels', '0.7.4', {
+        'checksums': ['7e77ead4619a3e11ab3c41982c8ad5b86edffe37c87fd2a37ec3c2cc6470b98a'],
+    }),
+    ('efficientnet_pytorch', '0.7.1', {
+        'checksums': ['00b9b261effce59d2d47aae2ad238c29a2a65175470f41ada7ecac439b7c1ee1'],
+    }),
+    ('huggingface_hub', '0.15.1', {
+        'checksums': ['a61b7d1a7769fe10119e730277c72ab99d95c48d86a3d6da3e9f3d0f632a4081'],
+    }),
+    ('timm', '0.9.2', {
+        'checksums': ['d0977cc5e02c69bda979fca8b52aa315a5f2cb64ebf8ad2c4631b1e452762c14'],
+    }),
+    ('tqdm', '4.64.0', {
```

Comment:
please use `tqdm/4.64.0-GCCcore-11.3.0` dependency instead?

Reply:
tqdm is a dependency on the new commit